src/status_im/ui/screens/wallet/main/views.cljs

```diff
@@ -48,9 +48,18 @@
                     :custom-content [toolbar-title]
                     :custom-action  [toolbar-buttons]}])
 
+(defn error-message-view [error-message]
+  (when error-message
+    [rn/view {:style st/wallet-error-container}
+     [rn/view {:style st/wallet-exclamation-container}
+      [vi/icon :icons/exclamation_mark {:color :white
+                                        :container-style st/wallet-error-exclamation}]]
+     [rn/text {:style st/wallet-error-message} (i18n/label :t/wallet-error)]]))
+
 ;; TODO(oskarth): Whatever triggers the "in progress" animation should also trigger wallet-init or load-prices event.
-(defn main-section [usd-value change]
+(defn main-section [usd-value change error-message]
   [rn/view {:style st/main-section}
+   [error-message-view error-message]
```

Comment:
Why not put the conditional here instead of in `error-message-view` ? Looks easyer to read to me.